### PR TITLE
add merge import declaration refactor

### LINKED ISSUE #26487: quick fix for merge duplicate import or export declaration
## Search Terms
quickfix, import, export

## Suggestion

provide a quickfix for merge two(or more)  import or export declaration

## Use Cases

it's useful for resolve git conflict 

## Examples
```ts
// ====
import { a, b, c } from 'mod'
// ====
import { b, d, e } from 'mod'
// ====
```

keep each other for git conflict:

```ts
import { a, b, c } from 'mod'
import { b, d, e } from 'mod'
```

after merge:
```ts
import { a, b, c, d, e } from 'mod'

```

## Checklist

My suggestion meets these guidelines:
* [x] This wouldn't be a breaking change in existing TypeScript / JavaScript code
* [x] This wouldn't change the runtime behavior of existing JavaScript code
* [x] This could be implemented without emitting different JS based on the types of the expressions
* [x] This isn't a runtime feature (e.g. new expression-level syntax)

### PR DESCRIPTION
**Pull request by Kingwl**
Fixes #26487

### CODE REVIEW

File: src/services/refactors/mergeImportDeclaration.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* @internal */
+namespace ts.refactor.addOrRemoveBracesToArrowFunction {
```
Comment (ajafff) [inline src/services/refactors/mergeImportDeclaration.ts]: That name should probably be amended

**Comment by RyanCavanaugh** [comment]
@uniqueiniquity @amcasey can you review the refactoring code please?

We should also figure out better phrasing; I don't like "duplicate" here.

**Comment by Kingwl** [comment]
>  I don't like "duplicate" here.

`conflict`?🙋🏻‍♂️

**Comment by amcasey** [comment]
Would this be more useful as a code fix?  It doesn't seem like it would be excessively noisy and the transformation is one-way so the suggestion wouldn't immediately be replaced by its opposite.

**Comment by amcasey** [comment]
Does this do something that Organize Imports doesn't do?  If so, should Organize Imports be enhanced?

**Comment by Kingwl** [comment]
> Would this be more useful as a code fix? It doesn't seem like it would be excessively noisy and the transformation is one-way so the suggestion wouldn't immediately be replaced by its opposite

Yes, but How could we provide the fix all action in this case?, maybe there are 

> Does this do something that Organize Imports doesn't do? If so, should Organize Imports be enhanced?

IMO: the error should be fix by a explicit behavior

**Comment by Kingwl** [comment]
⬆️

**Comment by Kingwl** [comment]
⬆️ again,
And i remember that why i donot use the quickfix: 
import from same module is not an error

**Comment by Kingwl** [comment]
seems already existed

**Pull request closed without merging**